x-pack/plugins/monitoring/server/lib/metrics/apm/classes.js

```diff
@@ -69,7 +69,7 @@ export class ApmCpuUtilizationMetric extends ApmMetric {
           metricDerivNormalizedValue >= 0 &&
           metricDerivNormalizedValue !== null
         ) {
-          return metricDerivNormalizedValue / bucketSizeInMillis * 100;
+          return metricDerivNormalizedValue / bucketSizeInMillis * 1000;
```

Comment:
I'm not sure this is correct but it's possible I'm following this code incorrectly. My understanding is that we were multiplying by 100 here to get to a percentage value. As for the conversion to `ms`, I don't know if that's necessary because isn't `metricDerivNormalizedValue` in `ms` already?

Reply:
It's a good point. Hopefully @tsullivan can provide some clarification here.

I actually assumed it was this based on the formula [you used here](https://github.com/elastic/beats/issues/9377#issuecomment-446315211):
> 68695.0 / (30 * 1000) = 2.28 = 228%

Reply:
What units are `metricDerivNormalizedValue` in? That's the answer here.

> 68695.0 / (30 * 1000) = 2.28 = 228%

That is correct because `30` is in `s` and `68695` is in `ms`, so you need to convert one or the other to erase the units to get down the percentage.

Reply:
I assumed the `metricDerivNormalizedValue` is in ms (which comes from `beats_stats.metrics.beat.cpu.total.value`) - can you verify @ycombinator?

Reply:
The unit of `beats_stats.metrics.beat.cpu.total.value` is `ms`.

Reply:
The bucket size is in `s` so the math here is correct then?